### ISSUE TITLE
Specifying --password seems to have no effect

### ISSUE THREAD

**Issue by grwlf** [Evidlo/remarkable_mouse#63]
Hi. I'm trying 24ca4f6fc4ef8d13edff1bbd7c480ee9f9a38fe9 and the following command line:

```
remouse --password=MyPaSsWoRd --mode=stretch --orientation=left
```

The password prompt appears despite the fact that I already specified it using the command line. Entering it again interactively works fine (the gui window appears). Could it be the second SSH connection which doesn't reuse password or smth other?

**Comment by alicuza** [comment]
Same for me, the password command doesn't seem to make a difference.

**Comment by SBAPKat** [comment]
this is due to the password variable being reset to none in remarkable_mouse.py line 40, removing that line and rebuilding/reinstalling does the trick

[https://github.com/Evidlo/remarkable_mouse/blob/6ed83f14cbfa996be2d7a6463ba68b81d041eb64/remarkable_mouse/remarkable_mouse.py#L40](url)

**Comment by Evidlo** [comment]
Should be fixed in 7.0.0.
